projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -114,6 +112,14 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
                 </ng-container>
               </li>
             </ng-container>
+            <li class="accordion-list-item row text-break">
+              <div class="col item-label">
+                {{ 'quote.totalPrice' | cxTranslate }}
+              </div>
+              <div class="col notice" *ngFor="let cart of quotes?.carts">
+                {{ cart.totalPrice?.formattedValue }}
```

Comment:
Same here: Did you check if the same structure is for banking products?

Reply:
Yes, this structure works with banking products